api/cas-server-core-api-configuration-model/src/main/java/org/apereo/cas/configuration/model/support/oauth/OAuthProperties.java

```diff
@@ -36,6 +37,14 @@
      */
     private boolean replicateSessions;
 
+    /**
+     * The pac4j framework sets a CSRF cookie as part of its Oauth configuration.
```

Comment:
This also seems way too specific. Let's not refer to the specific choice of framework and its behavior if possible. If that behavior changes, it's not possible to catch and track this change and update it. It would be preferable to describe what this setting does in generic terms without tying it to a specific implementation, etc. Note that such comments show up in the docs. The average user has no clue what generally "pac4j" is. Something more generic would be preferable like "Control or adjust the CSRF cookie settings  in OAUTH authentication flows".

Reply:
I think I still refer to pac4j defaults in one place but it's now in the constructor javadoc which I don't think shows up in the docs (other than in the javadocs). I feel like there needs to be someplace to put comments to other developers that might be looking at the code in the future.